emails/utils.py

```diff
@@ -447,15 +449,29 @@ def count_all_trackers(html_content):
     )
 
 
-def remove_trackers(html_content, level="general"):
+def remove_trackers(html_content, from_address, datetime_now, level="general"):
     trackers = general_trackers() if level == "general" else strict_trackers()
     tracker_removed = 0
     changed_content = html_content
 
     for tracker in trackers:
         pattern = convert_domains_to_regex_patterns(tracker)
+
+        def convert_to_tracker_warning_link(matchobj):
+            quote, original_link, _ = matchobj.groups()
+            tracker_link_details = {
+                "sender": from_address,
+                "received_at": datetime_now,
+                "original_link": original_link,
+            }
+            anchor = urllib.parse.quote_plus(
```

Comment:
Nit: if you change the import above, this changes to
```suggestion
            anchor = quote_plus(
```